ui/constants/pages.js

```diff
@@ -24,6 +24,7 @@ exports.TAGS_FOLLOWING = 'following/tags';
 exports.TAGS_FOLLOWING_MANAGE = 'following/tags/manage';
 exports.CHANNELS_FOLLOWING = 'following/channels';
 exports.CHANNELS_FOLLOWING_MANAGE = 'following/channels/manage';
+exports.CHANNELS_FOLLOWING_DISCOVER = 'following/channels/discover';
```

Comment:
tempted to change all following URLs as follows:

/following/tags -> /tags (it's not directly used, just to prefix other tags, and https://lbry.tv/$/tags is semi-broken)
/following/tags/manage -> /tags/mange (or /tags/edit)
/following/channels -> /following
/following/channels/discover -> /following/discover
/following/channels/manage -> (gone)

If not to the above, to something that ensures:
- Anything in the right sidebar has only one word in the path (i.e. always `/$/foo`, never `/$/foo/bar`)
- Nothing has more than two words in the path

Reply:
Will update this in my following PR